translations/monero-core_it.ts

```diff
@@ -16,17 +16,17 @@
     <message>
         <location filename="../pages/AddressBook.qml" line="63"/>
         <source>&lt;b&gt;Tip tekst test&lt;/b&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Tip tekst test&lt;/b&gt;</translation>
```

Comment:
This should not be "translated" as it is not real text.
Will be removed in the future and marking it as finished will make that harder.